src/solvers/dgsem_structured/containers.jl

```diff
@@ -11,6 +11,8 @@ struct ElementContainer{NDIMS, RealT<:Real, uEltype<:Real, NDIMSP1, NDIMSP2, NDI
   node_coordinates      ::Array{RealT, NDIMSP2}   # [orientation, node_i, node_j, node_k, element]
   # ID of neighbor element in negative direction in orientation
   left_neighbors        ::Array{Int, 2}           # [orientation, elements]
+  # ID of neighbor element in positive direction in orientation
+  right_neighbors       ::Array{Int, 2}           # [orientation ,elements]
```

Comment:
Instead of left neighbors and right neighbors, I think it should just be neighbor ids. But this is for another PR, so feel free to close this :grinning: